### PR TITLE
MGMT-16450: Disable vsphere and nutanix periodic jobs until fixed, and decreasing the frequency of ocm-2.7 periodic jobs to twice a week

### PR DESCRIPTION
**Pull request by danmanor**
Disable `vsphere` and `nutanix` periodic jobs until fixed, and decreasing the frequency of `ocm-2.7` periodic jobs to twice a week

### CODE REVIEW

File: ci-operator/jobs/openshift/assisted-test-infra/openshift-assisted-test-infra-master-presubmits.yaml [hunk 1]
```diff
@@ -3130,249 +3130,6 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-single-node-with-worker-live-iso,?($|\s.*)
-  - agent: kubernetes
```
Comment (adriengentil) [inline ci-operator/jobs/openshift/assisted-test-infra/openshift-assisted-test-infra-master-presubmits.yaml:3133]: leave the presubmits

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml [hunk 2]
```diff
@@ -683,87 +683,6 @@ tests:
         OPENSHIFT_VERSION=4.14
       PLATFORM: vsphere
     workflow: assisted-vsphere
-- as: e2e-vsphere-assisted-periodic
```
Comment (adriengentil) [inline ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:686]: don't we have vsphere/nutanix periodics in 2.8/2.9?

Comment (danmanor) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:686]: Yes my bad thanks

**Review by adriengentil — APPROVED** [review]

**Pull request merged**